modules/src/ics07_tendermint/client_state.rs

```diff
@@ -0,0 +1,82 @@
+use crate::ics02_client::client_type::ClientType;
+use crate::ics23_commitment::CommitmentRoot;
+
+use crate::ics07_tendermint::header::Header;
+use crate::ics24_host::client::ClientId;
+use serde_derive::{Deserialize, Serialize};
+use std::time::Duration;
+use tendermint::lite::Header as liteHeader;
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct ClientState {
+    id: String,
+    trusting_period: Duration,
```

Comment:
Note (mostly to myself): 

in the go code, time.Duration actually is this type alias:
```go
// A Duration represents the elapsed time between two instants
// as an int64 nanosecond count. The representation limits the
// largest representable duration to approximately 290 years.
type Duration int64
```
while rust's `std::time::Duration` actually is 
```rust
pub struct Duration {
    secs: u64,
    nanos: u32, // Always 0 <= nanos < NANOS_PER_SEC
}
```
Same for the `unbonding_period` field below. 

So for deserialization (independent from amino or not) we need to translate between the two.